crates/re_types_builder/src/codegen/rust/reflection.rs

```diff
@@ -21,38 +23,48 @@ pub fn generate_reflection(
 
     let mut imports = BTreeSet::new();
     let component_reflection = generate_component_reflection(reporter, objects, &mut imports);
+    let archetype_reflection = generate_archetype_reflection(reporter, objects);
 
     let mut code = format!("// {}\n\n", autogen_warning!());
-    code.push_str("#![allow(unused_imports)]\n");
+    code.push_str("#![allow(clippy::too_many_lines)]\n");
```

Comment:
huh, news to me that we have this lint active anywhere

Reply:
Good! It means you don't often hit our very high function line limit

https://github.com/rerun-io/rerun/blob/7b359381a987b5efc866cc9b3ef747ec9db003f4/clippy.toml#L24